File: Python/Search_Insert_Position.py
class Solution:
    def searchInsert(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: int
        """
        index = 0
        find = False
        while index < len(nums) and find == False:
            if target > nums[index]:
                index += 1
            else :
                find = True
        return index

File: C++/Search_Insert_Position.cpp
class Solution {
public:
    int searchInsert(vector<int>& nums, int target) {
        auto p = begin(nums);
        for(; p != end(nums); p++) {
            if (target <= *p) {
                return (p-begin(nums));
                break;
            }
        }
        return (p-begin(nums));
    }
};